otelcollector/deploy/addon-chart/azure-monitor-metrics-addon/templates/ama-metrics-daemonset.yaml

```diff
@@ -461,9 +462,11 @@ spec:
              cpu: 100m
              memory: 100Mi
           securityContext:
-           capabilities:
-             add:
-               - NET_ADMIN
+            capabilities:
+              drop:
+                - ALL
+              add:
+                - NET_ADMIN
```

Comment:
Hi Rashmi - so i see that our container (prom collector) for both DS & RS, it needs DAC_OVERRIDE, and token adapter container needs NET_ADMIN . I also see linux token adapter need NET_RAW (in addition to NET_ADMIN), but only for linux and not for windows ?

For KSM & TA, i see we drop all, which is great.